DEVELOPMENT.md

```diff
@@ -8,12 +8,11 @@ use [minikube](https://github.com/kubernetes/minikube):
 MacOS:
 
 ```bash
-brew install docker-machine-driver-hyperkit
-sudo chown root:wheel /usr/local/opt/docker-machine-driver-hyperkit/bin/docker-machine-driver-hyperkit
-sudo chmod u+s /usr/local/opt/docker-machine-driver-hyperkit/bin/docker-machine-driver-hyperkit
+brew install minikube
+brew install hyperkit
 
-brew cask install minikube
-minikube config set vm-driver hyperkit
+minikube start --driver=hyperkit
```

Comment:
Perhaps, this line should not be here — the cluster is started a few lines below in the next snippet.